user_profile/views.py

```diff
@@ -0,0 +1,242 @@
+import json
+from django.contrib import messages
+from django.core import serializers
+from django.http import JsonResponse
+from django.shortcuts import render,redirect
+from django.contrib.auth.mixins import LoginRequiredMixin
+from django.contrib.auth import update_session_auth_hash
+from django.views import View
+from django.views.generic import TemplateView,ListView
+from django.db.models import Q
+from accounts.models import CustomUser
+from . import models,forms
```

Comment:
Please use absolute imports throughout the app.

Reply:
fixed